src/agora/docoding.cpp

```diff
@@ -75,6 +83,8 @@ EventData DoEncode::Launch(size_t tag) {
   if (CyclesToUs(duration, cfg_->FreqGhz()) > 500) {
     std::printf("Thread %d Encode takes %.2f\n", tid_,
                 CyclesToUs(duration, cfg_->FreqGhz()));
+  
+  std::free(scramble_buffer_ptr);
```

Comment:
See comment above.  If we move to new, change this to delete. Otherwise it is fine.

Reply:
Yes, will change accordingly.